core/services/chainlink/application.go

```diff
@@ -335,20 +337,28 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	}
 	if cfg.FeatureOffchainReporting2() {
 		globalLogger.Debug("Off-chain reporting v2 enabled")
-		relayers := make(map[relaytypes.Network]relaytypes.Relayer)
+		relayers := make(map[relay.Network]relaytypes.Relayer)
 		if cfg.EVMEnabled() {
 			evmRelayer := evmrelay.NewRelayer(db, chains.EVM, globalLogger.Named("EVM"))
-			relayers[relaytypes.EVM] = evmRelayer
+			relayers[relay.EVM] = evmRelayer
 			subservices = append(subservices, evmRelayer)
 		}
 		if cfg.SolanaEnabled() {
-			solanaRelayer := pkgsolana.NewRelayer(globalLogger.Named("Solana.Relayer"), chains.Solana, keyStore.Solana())
-			relayers[relaytypes.Solana] = solanaRelayer
+			// TODO: Goes away with solana txm
```

Comment:
will this go away? right now the tx signing still happens in the relay even with the txm...

is the expectation the txm does the transaction signing?

Reply:
update: made a story to make this change in the txm work 👍🏼

Reply:
It is already in progress here: https://github.com/smartcontractkit/chainlink/pull/6445